scripts/fetch_acs_medicaid.py

```
@@ -0,0 +1,260 @@
+    return f"{API_BASE}/{year}/acs/acs5?get={get_clause}&{for_clause}"
+
+
+def fetch_one(year: int, geo: str, api_key: str, timeout: int = 120) -> pd.DataFrame:
+    url = build_url(year, geo) + f"&key={api_key}"
+    # Don't print the key.
+    safe = url.replace(api_key, "***")
+    print(f"  GET {safe}", flush=True)
+
+    resp = requests.get(url, timeout=timeout)
+    if resp.status_code == 404:
+        raise FileNotFoundError(f"{year} {geo}: 404 (table or year not published yet)")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)